recipes/apache-flink/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "apache-flink" %}
```

Comment:
This might not be the right name to use if this is primarily targeting the Python wrapper.  Maybe `pyflink` instead?